mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/jobcluster/job/MantisJobMetadataImpl.java

```diff
@@ -503,4 +503,10 @@ public int hashCode() {
     public Costs getJobCosts() {
         return jobCosts;
     }
+
+    @Override
+    public Optional<String> getStageAttribute(int stageNum, String sizeName) {
```

Comment:
do you mean sizeNameKey?

Reply:
I will rename this as `getStageAttributeByKey`